testData/solutions/win32.json5

```diff
@@ -7628,22 +7738,40 @@
                             }
                         }
                     },
+                    // GPII-4207 sgithens speed
+                    // Having tested this in conjuction with the control panel it seems the
+                    // max and min values are: (what is shown when you hover over the slider)
+                    //
+                    //           Control Panel UI        GPII Capture
+                    // low:      10                      10
+                    // high:     358                     358
+                    // default:  80                      80
                     "MaxSpeed": {
                         "schema": {
                             "title": "Mouse keys speed",
                             "description": "Speed of mouse keys",
                             "type": "number",
-                            "multipleOf": 10
+                            "multipleOf": 10,
+                            "default": 80
                         }
                     },
+                    // GPII-4207 sgithens acceleration
+                    // Having tested this in conjuction with the control panel it seems the
+                    // max and min values are: (what is shown when you hover over the slider)
+                    //
+                    //           Control Panel UI        GPII Capture
+                    // low:      100                     5000
+                    // high:     500                     1000  (yes it's inverse)
+                    // default:  300                     3000
                     "Acceleration": {
                         "schema": {
                             "title": "Mouse keys acceleration",
                             "description": "Acceleration of mouse keys",
                             "type": "number",
```

Comment:
All the examples make it seem like this should be type `integer` instead of `number`?

Reply:
That seems like it would be correct. Have updated it to `integer`